toontown/spellbook/MagicWordIndex.py

```diff
@@ -287,30 +287,33 @@ def handleWord(self, invoker, avId, toon, *args):
 
         return f"Successfully maxed {toon.getName()}!"
     
-class RestockInventory(MagicWord):
-    aliases = ['allstuff', 'restockinv', 'maxinv', 'maxinventory', 'restock']
-    desc = 'Gives target all the inventory they can carry.'
+class Inventory(MagicWord):
+    # by default restock the inventory
+    aliases = ['gags', 'inv']
+    desc = 'This allows you to modify your inventory in various ways.'
     execLocation = MagicWordConfig.EXEC_LOC_SERVER
-
+    arguments = [("command", str, False, ''), ("track", str, False, ""), ("level", int, False, 0), ("amount", int, False, 0)]
```

Comment:
Command should probably be marked as required.
```suggestion
    arguments = [("command", str, True, ''), ("track", str, False, ""), ("level", int, False, 0), ("amount", int, False, 0)]
```

Reply:
No littlecat said he wanted the default to restock